setup.cfg

```diff
@@ -52,6 +52,7 @@ net =
   python-dateutil
   zeep
   tqdm
+  parse
```

Comment:
I wonder if we don't actually add this as a dependency by copy the file to the extern folder instead.